docs/installation/introduction.rst

```diff
@@ -1,6 +1,104 @@
 Introduction
 ============
 
+Foreword
+^^^^^^^^
+MetalK8s is a Kubernetes_ distribution with a number of addons carefully
+picked for optimal on-premises deployments, including pre-configured monitoring
+and alerting, self-healing system configuration, and more.
+
+The installation of a MetalK8s cluster can be broken down into
+the following steps:
+
+#. :doc:`Setup <./setup>` of the environment (with requirements and example
+   OpenStack deployment)
+#. :doc:`Deployment <./bootstrap>` of the :term:`Bootstrap node`, the first
+   machine in the cluster
+#. :doc:`Expansion <./expansion>` of the cluster, orchestrated from the
+   Bootstrap node
+
+.. _Kubernetes: https://kubernetes.io/
+
+.. _installation-intro-architecture:
+
+Choosing a Deployment Architecture
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before starting the installation, choosing an architecture is recommended, as
+it can impact sizing of the machines and other infrastructure-related details.
+
+.. note:: "Machines" may indicate bare-metal servers or VMs interchangeably
+
+Standard Architecture
+"""""""""""""""""""""
+MetalK8s is designed with on-premise, offline, reliable deployments in mind.
+The first example architecture proposed follows this spirit, focusing on
+reliability rather than compacity:
+
+- One machine dedicated to running Bootstrap services (see
```

Comment:
When deploying the bootstrap node, it automatically deploys services related to master or etcd so achieving this architecture would mean to remove some roles/taints, it is really the standard arch we want to recommend? shouldn't we suggest a standard arch where bootstrap node is also having matser/etcd role?

Reply:
I think we prefer running bootstrap services independently, to prevent Salt Master from having an impact on control plane services. The fact that Bootstrap runs everything after the initial installation is just a constraint (you need it to start expanding the cluster). So I don't know, maybe I can reword the "standard" into something like "extended" / "flat". WDYT?

Reply:
I would like to have @xaf-scality opinion on this one ...

Reply:
Had a face to face conversation with him: we'll use the 3-node control-plane + infra, including bootstrap, as the "standard", and suggest the "extended" as a third option.